src/handlers.js

```diff
@@ -65,6 +65,27 @@ async function singlePackageListing(req, res, timecop) {
       og_image_height: 600,
     }});
   } catch(err) {
+    let status_to_display = false; // Since the status is ignored if not a number,
+    // we initialize as boolean to no-op in the case we don't find a proper status
+
+    const validStatusIs = (val, key) => {
+      if (val?.response?.[key] && typeof val.response[key] === "boolean" && val.response[key]) {
```

Comment:
```suggestion
      if (typeof val?.response?.[key] === "boolean" && val.response[key]) {
```

On second thought, maybe this, and it does sort of match with `  if (typeof details?.status_to_display === "number") {` in the other file now.

Reply:
It's been a while since looking closely at this code.
But if memory serves, as of now, this should always be present, and there wouldn't be a case where it's fully undefined, but with that said, the idea is that in the future it may be totally possible for `val` to be totally undefined, in which case we might as well protect from it now, rather than let the problem sneak up on us.

So thanks for finding that flaw, and lets add this in so we are good to merge as soon as possible, thanks again for the awesome feedback on this one!

Reply:
INFORMATIONAL ONLY NOTE, NO ACTION NEEDED:

I am sad to report, after testing in the node REPL, that `typeof val?.response?.[key]` will error if `val` is not defined :/

`Uncaught ReferenceError: val is not defined`

So, this was not really an error safety thing after all, and one would have to `try/catch` if not throwing was important for `val` not being defined.

ONCE AGAIN< INFORMATIONAL ONLY< NO ACTION NEEDED, just thought I should share what I found in actual testing.